### DISCUSSION TITLE
Create catalog catalog_name ?

### DISCUSSION THREAD

**Discussion by bladko-tom** [duckdb/duckdb#8601] [Ideas]
Do we plan this feature ? Is it technically possible ?

**Comment by Mause** [comment]
Catalogs are full databases, you can create new ones by attaching a new in memory or on disk database using the ATTACH statement

**Reply by bladko-tom** [reply]
i have large data lake with parquets in different locations. All are registered as views on duckdb with parquet reader. This of course works great with duckDB :) But structure is complex and would be great to have something on the top of schema / view like catalog not to confuse users where to find what..

problem is that a lot of views (not raw data, but output views) are based on other views, which makes hard to split between connections / databases as view cannot read data from another database only from same database

how to resolve this limitation ? ability to create catalog here would be great but i do understand this is design of DuckDB ?

![image](https://github.com/duckdb/duckdb/assets/112213846/0792d879-77b6-43ea-8ba7-be91f89383a0)
